src/bandwidthRtc.ts

```diff
@@ -1,11 +1,14 @@
-require("webrtc-adapter");
+if (globalThis.window) {
+  require("webrtc-adapter");
+}
```

Comment:
Turns out `webrtc-adapter` uses `window` a lot, and it doesn't exist in a node runtime; makes sense.